site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -11,11 +10,17 @@ import { useTranslation } from "react-i18next"
 import { getFormHelpers, nameValidator, onChangeTrimmed } from "util/formUtils"
 import * as Yup from "yup"
 import { AlertBanner } from "components/AlertBanner/AlertBanner"
-import { makeStyles } from "@material-ui/core/styles"
 import { FullPageHorizontalForm } from "components/FullPageForm/FullPageHorizontalForm"
 import { SelectedTemplate } from "./SelectedTemplate"
 import { Loader } from "components/Loader/Loader"
 import { GitAuth } from "components/GitAuth/GitAuth"
+import {
+  FormFields,
+  FormSection,
+  FormFooter,
+  HorizontalForm,
+} from "components/HorizontalForm/HorizontalForm"
+import { makeStyles } from "@material-ui/core/styles"
```

Comment:
I think it's probably time for this file to be broken apart.

Reply:
I think it is! I will do that.